.vscode/extensions.json

```diff
@@ -0,0 +1,6 @@
+{
+    "recommendations": [
+        "eg2.tslint",
```

Comment:
Why is TSLint a recommended package for openHAB? There's no *.ts files in OH configuration folder.

Reply:
This is shown, when you are working *on the extension itself*.
So "recommended for developing the extension"

Reply:
If we would want somethign for openHAB itself we would have to add an `.vscode` folder with that file in the OH configuration folder.

Reply:
Fair enough! Thanks for explanation ;)